src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/AdjudicationsResourceTest.java

```diff
@@ -923,7 +923,7 @@ public void createSanctionsReturnsSuccess() {
             assertThat(oicSanctions.get(0).getOffenderBookId()).isEqualTo(-50L);
             assertThat(oicSanctions.get(0).getSanctionSeq()).isEqualTo(1L);
             assertThat(oicSanctions.get(0).getOicSanctionCode()).isEqualTo(OicSanctionCode.ADA);
-            assertThat(oicSanctions.get(0).getCompensationAmount()).isEqualTo(new BigDecimal("1000.55"));
+            assertThat(oicSanctions.get(0).getCompensationAmount()).isEqualByComparingTo("1000.55");
```

Comment:
this switches to using the [BigDecimal](https://joel-costigliola.github.io/assertj/core-8/api/org/assertj/core/api/AbstractBigDecimalAssert.html) assert